src/server/polyfill/intl.js

```diff
@@ -14,4 +14,8 @@
  * permissions and limitations under the License.
  */
 
-global.Intl = require('lean-intl');
+import Intl from 'lean-intl';
+
+if (!process.env.ONE_CONFIG_USE_NATIVE_POLYFILL) {
```

Comment:
```suggestion
if (process.env.ONE_CONFIG_USE_NATIVE_POLYFILL !== 'true') {
```

Reply:
Good catch! I have fixed this in [d76bbe5](https://github.com/americanexpress/one-app/pull/1258/commits/d76bbe516d1724bdd802457057f5727466429a71)